Common/itkTransformixInputPointFileReader.hxx

```diff
@@ -40,30 +40,60 @@ TransformixInputPointFileReader<TOutputMesh>::GenerateOutputInformation()
   {
     this->m_Reader.close();
   }
-  this->m_Reader.open(this->m_FileName);
 
-  /** Read the first entry */
-  std::string indexOrPoint;
-  this->m_Reader >> indexOrPoint;
+  if( this->m_Binary ) {
```

Comment:
We use automatic code formatting by clang-format. It looks like this code still needs to be formatted that way, as I would expect it to place the `{` on the next line. (We use a specific (old) version of clang-format: [8.0.1](https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1/folder/5d635cbcd35580e6dcbd80b6)) No problem, of course, we can fix that later.